week2/homework/src/index.js

```diff
@@ -1,3 +1,126 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const chalk = require('chalk');
+const yargs = require('yargs');
+const rl = require('readline').createInterface(process.stdin, process.stdout);
```

Comment:
It's not always needed the use case of `readline`, if I ran the command `node index.js` with out any other command it will stuck.

Also line 22 you are just closing it.
so you may create interface on every handler or you could create your own function using `Promise` and async await on every handler, it will be a good exercise :)